src/vs/workbench/contrib/terminal/common/terminal.ts

```diff
@@ -102,7 +102,7 @@ export interface ITerminalConfiguration {
 		windows: string[];
 	};
 	profiles: ITerminalProfiles;
-	detectWslProfiles: boolean;
+	quickLaunchWslProfiles: boolean;
```

Comment:
It's getting long, but maybe `showQuickLaunchWslProfiles` is better as the name kind of looks like you would define profiles (instead of true/false).